src/ensembl/src/content/app/entity-viewer/gene-view/components/collapsed-exons-image/CollapsedExonsImage.tsx

```diff
@@ -76,28 +71,22 @@ export const ExonsImage = (props: ExonsImageProps) => {
 const ExonsImageWithData = (props: ExonsImageWithDataProps) => {
   const { transcript } = props;
 
-  if (!transcript.cds) {
+  if (!transcript.product) {
     return null;
   }
 
-  const codingExons = getCodingExonsForImage(transcript);
-
-  const getSplicedRNALength = () => {
-    const rnaLength = transcript.exons.reduce((length, exon) => {
-      const { start, end } = getFeatureCoordinates(exon);
-      return length + (end - start + 1);
-    }, 0);
-
-    return getCommaSeparatedNumber(rnaLength);
-  };
-
+  // If we consider the image's starting point as A and end point as B the scale can be sketched as:
+  // A--  longest protein's length  --B
+  // A--      width in pixels       --B
```

Comment:
This comment isn't really explaining what's going on. It's just repeating what we see two lines below in:

```
.domain([0, props.longestProteinLength])
.range([0, props.width])
```

Maybe something like this:

> Create a scale where the domain is the total length of the track, in amino acids.
> The track is as wide as the longest protein generated from the gene;
> therefore, it is guaranteed that the length of the protein drawn by this component
> will fall within this domain

Reply:
✔️